assets/webpack.config.js

```diff
@@ -71,48 +109,76 @@ module.exports = (env, argv) => {
                   "@babel/preset-react",
                   "@babel/preset-typescript",
                 ],
-                plugins: [
-                  "@babel/plugin-proposal-export-default-from",
-                  "@babel/plugin-proposal-logical-assignment-operators",
-                  [
-                    "@babel/plugin-proposal-optional-chaining",
-                    { loose: false },
-                  ],
-                  [
-                    "@babel/plugin-proposal-pipeline-operator",
-                    { proposal: "minimal" },
-                  ],
-                  [
-                    "@babel/plugin-proposal-nullish-coalescing-operator",
-                    { loose: false },
-                  ],
-                  "@babel/plugin-proposal-do-expressions",
-                ],
+                plugins: common_babel_loader_plugins,
               },
             },
           },
+          ...common_rules,
           {
-            enforce: "pre",
-            test: /\.js$/,
-            loader: "source-map-loader",
-          },
-          {
-            test: /\.s?css$/,
+            test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
```

Comment:
I wonder if we could also de-duplicate this rule, since the DUP config only has one parameter that's different. Maybe a little helper function?